### DISCUSSION TITLE
Confirm Form Resubmission Error

### DISCUSSION THREAD

**Discussion by yodaphone** [benbusby/whoogle-search#890] [Q&A]
Just installed whoogle and want to know how to avoid the "Confirm Form Resubmission" error when i hit back button on the search page

**Comment by ozhound** [comment]
> Just installed whoogle and want to know how to avoid the "Confirm Form Resubmission" error when i hit back button on the search page

I too would like a solution to this

**Comment by binitGajera** [comment]
Feel free to check this issue out: #1026.

Basically using POST requests seems to be causing this, the issue would get resolved when setting the configuration variable `WHOOGLE_CONFIG_GET_ONLY` to `1`.
